cli.py

```diff
@@ -38,12 +39,26 @@ def unstar_channel(channel_id):
     r = requests.get(url, params=params)
     r.raise_for_status()
 
+def get_muted_channels():
+    url = BASE_URL + "users.prefs.get"
+    params = {"token": TOKEN}
+    r = requests.get(url, params=params)
+    r.raise_for_status()
+    data = json.loads(r.text)
+    return data["prefs"]["muted_channels"]
```

Comment:
If `"muted_channels"` does not exist (e. g. the response format changed), this will throw an exception. Do we want to handle this?

Reply:
Thinking about it, we also don't handle request exceptions. So let's not bother in this PR.